src/app/shared/services/repositories/proof/proof.ts

```diff
@@ -7,6 +7,7 @@ import { ImageStore } from '../../image-store/image-store.service';
 
 export class Proof {
   willCollectTruth = false;
+  diaBackendAssetId = '';
```

Comment:
I believe using `undefined` instead of `''` empty string could be safer as TypeScript prefers [Nullish Coalescing](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing), which won't work if we use empty string as default value.

Also, `IndexedProofView` use the same nullable property.